src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,120 @@
-export const Pagination = () => {};
+import React, { MouseEvent } from 'react';
+
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  totalItems: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (value: number) => void,
+};
+
+export const Pagination: React.FC<Props> = ({
+  totalItems,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const items = getNumbers(1, totalItems)
+    .map(n => `Item ${n}`);
+
+  const pagesCount = Math.ceil(items.length / perPage);
+  const pages = getNumbers(1, pagesCount);
+  const fromItem = currentPage * perPage - perPage + 1;
+  const toItem = (currentPage * perPage) > totalItems
+    ? totalItems
+    : currentPage * perPage;
+
+  const handlePageSelector = (event: MouseEvent<HTMLElement>) => {
+    const value = (event.target as HTMLElement).textContent;
+
+    if (value) {
+      switch (value) {
+        case '«':
+          // eslint-disable-next-line
+          currentPage > 1
+            ? onPageChange(currentPage - 1)
+            : '';
+          break;
+
+        case '»':
+          // eslint-disable-next-line
+          currentPage < pagesCount
+            ? onPageChange(currentPage + 1)
+            : '';
+          break;
+
+        default:
+          onPageChange(+value);
+          break;
+      }
+    }
+  };
+
+  const prevClass = cn({
+    'page-item': true,
+    disabled: currentPage === 1,
+  });
+
+  const nextClass = cn({
+    'page-item': true,
+    disabled: currentPage === pagesCount,
+  });
+
+  return (
+    <>
+      {/* eslint-disable-next-line */}
```

Comment:
Try to fix such error and remove comment
If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review